roles/wildfly_systemd/tasks/selinux_config.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Set PID file location
+  ansible.builtin.set_fact:
+    wildfly_pid_file_location: "{{ pid_file_location | default('/var/run') }}"
+
+- name: Ensure PID directory exists
+  ansible.builtin.file:
+    path: "{{ wildfly_pid_file_location }}"
+    state: directory
+    mode: '0755'
+
+- name: Set SELinux context for custom PID file location
+  ansible.builtin.file:
+    path: "{{ wildfly_pid_file_location }}/wildfly.pid"
```

Comment:
That's broken. First, the name of the file is set in the templates of the wildfly_systemd and second the user needs to be able to override the name of the file (without modifying the content of the collection).